src/index.js

```diff
@@ -2,7 +2,24 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import 'semantic-ui-css/semantic.min.css';
 
-import './sass/index.sass';
+import { Provider } from 'react-redux';
```

Comment:
What does the provider do?

Reply:
The provider wraps around the root component _(in this case the router)_ and provides access to the store to every other component where needed. You can check the [documentation](https://react-redux.js.org/api/provider) on it if you want more info on how it works.